src/util/StringToAddress.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+library StringToAddress {
```

Comment:
Did you take this from somewhere? We should have test cases for it.

Reply:
I took it from somewhere and enhanced it to support uppercase as well. Will add now.